plugins/ui/setup.cfg

```
@@ -28,6 +28,7 @@ install_requires =
     deephaven-core>=0.37.0
     deephaven-plugin>=0.6.0
     json-rpc
+    pyjsonpatch
```

We should pin json-rpc and pyjsonpatch. Right now if either of these libraries update with a breaking change, users with new installs may not work correctly.

```suggestion
    json-rpc>=1.15.0
    pyjsonpatch>=0.1.1
```

Docs on compatible release notation: https://peps.python.org/pep-0440/#compatible-release